data/books.json

```diff
@@ -14,7 +14,8 @@
         "rating": 5,
         "subtitle": "ES6 & Beyond. It covers all aspects of javaScript deep down.",
         "topic": "JavaScript",
-        "website": "https://github.com/getify/You-Dont-Know-JS/tree/master/es6%20&%20beyond"
+        /* "website": "https://github.com/getify/You-Dont-Know-JS/tree/master/es6%20&%20beyond" */
```

Comment:
@Pavan-Kamthane Remove the commented line.

Reply:
Deleted